src/asciidocParser.ts

```diff
@@ -284,9 +286,9 @@ export class AsciidocParser {
       adocCmdArgs.push('-a', 'env-vscode')
 
       adocCmdArgs.push('-q', '-B', '"' + baseDir + '"', '-o', '-', '-')
-      const asciidoctor = spawn(adocCmd, adocCmdArgs, options)
```

Comment:
We should probably rename this variable to `asciidoctorProcess` to make it clear that we are dealing with the Asciidoctor CLI process.